src/contexts/SettingsProvider/SettingsProvider.js

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useEffect, useState } from 'react'
+
+const SettingsContext = createContext()
+
+const SettingsProvider = ({ value: propValue, children }) => {
+  const [networkType, setNetworkType] = useState(
+    localStorage.getItem('networkType') || 'mainnet',
+  )
+
+  useEffect(() => {
+    try {
+      const storedNetworkType = localStorage.getItem('networkType')
```

Comment:
It is not necessary to be done now, but would be nice to have a `// TODO` here so we do not forget to adjust it in the future.

`// TODO: Localstorage is being used directly in the Provider. It would be nice to have an Entity for this dataset, as we have for user, and use a Service for LocalStorage as we have for IndexedDB` 

This will be better to isolate the local storage dependency, in case there is a bug or we want to change it we have just one place to work. And create a proper place(Entity) for future and more complex settings.